Core/clim-core/panes.lisp

```diff
@@ -651,17 +651,18 @@ order to produce a double-click")
                                   &key width height)
   (declare (ignore width height))
   ;; merge user specified options.
-  (let ((sr (call-next-method)))
+  (let ((sr (ignore-errors (call-next-method))))
```

Comment:
I haven't really looked into this but `ignore-errors` seems like sweeping the actual problem under the rug.

Is this change required for the pull request? If not, I would prefer to move it into a separate pull request.

Reply:
it did make one of demos not working, but the problem seems to be gone now. Removed commit. issue is on the tracker so if someone tackles it properly it will be for the better.